com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/resources/jsweb/dmloader.js

```diff
@@ -887,7 +880,6 @@ var Module = {
 
     preSync: function(done) {
         if (Module.persistentStorage != true) {
-            Module._syncInitial = true;
```

Comment:
No need to duplicate flag here because we should check `_syncInitial` and `presitentStorage` where needed